README.md

```diff
@@ -0,0 +1,8 @@
+# Component library
```

Comment:
I expect the contents of this to change in the future - I just didn't want the root-level README to be missing.